tests/test_add_mds_to_cluster.py

```diff
@@ -43,35 +50,37 @@ def modify_fs(new_active_count):
     """
     with open(TEMP_YAML_FILE, 'r') as yaml_file:
         cephfs_obj = munchify(yaml.safe_load(yaml_file))
-    cephfs_obj.spec.metadataServer.activeCount = int(new_active_count)
+    cephfs_obj.spec.metadataServer.activeCount = new_active_count
     with open(TEMP_YAML_FILE, 'w') as yaml_file:
         yaml.dump(cephfs_obj.toDict(), yaml_file, default_flow_style=False)
     log.info(f"Change the active_count to {new_active_count}")
-    return OCP.apply(yaml_file=TEMP_YAML_FILE)
+    assert CEPHFS.apply(yaml_file=TEMP_YAML_FILE)
+    return True
 
 
 def delete_fs(fs_name):
     """
     Deleting a ceph FS
     """
     log.info(f"Deleting the file system")
-    stat = OCP.delete(TEMP_YAML_FILE)
+    stat = CEPHFS.delete(resource_name=fs_name)
     if CEPHFS_DELETED.format(cephfs_name=fs_name) in stat:
         return True
     return False
 
 
-def verify_fs_exist(fs_name):
+def verify_fs_exist(pod_count):
```

Comment:
The filesystem exists even after we delete the all MDS pods. Can we check for the name of the pod in the filesystem instead of pod_count. The pod name gets created along with the filesystem name, hence we could exactly know the pod has got created in the filesystem.
Please correct me if my understanding was wrong.

Reply:
I'm not deleting pods here but the cephfilesystem.
I'm checking for pod count because when deleting the cephfilesystem, I'd expect that all pods will be deleted as well